src/dxvk/imgui/dxvk_imgui.cpp

```diff
@@ -1238,39 +1301,46 @@ namespace dxvk {
       ImGui::SetCursorPosX(x + startX + (thumbnailSize - extent.x) / 2.f);
       ImGui::SetCursorPosY(y + (thumbnailSize - extent.y) / 2.f);
 
-      if (ImGui::ImageButton(pair.second.texID, extent)) {
-        const char* action;
-        if (data.find(pair.first) != data.end()) {
-          data.erase(pair.first);
-          action = "removed";
-        }
-        else {
-          data.insert(pair.first);
-          action = "added";
-        }
-
-        char buffer[256];
-        sprintf_s(buffer, "%s - %s %016llX\n", uniqueId, action, pair.first);
-        Logger::info(buffer);
+      if (ImGui::ImageButton(pair.second.texID, extent) && isListFiltered) {
+        toggleTextureSelection(pair.first, uniqueId, listRtxOption.textureSetOption->getValue());
       }
 
       if (ImGui::IsItemHovered()) {
         std::stringstream formatName;
         formatName << imageInfo.format;
 
+        //list all selections for this texture
+        std::string rtxTextureSelection;
+        for (auto& rtxOption : rtxTextureOptions) {
+          rtxOption.bufferTextureHash = pair.first;
+          rtxOption.bufferToggle = rtxOption.textureSetOption->getValue().find(pair.first) != rtxOption.textureSetOption->getValue().end();
+          if (rtxOption.bufferToggle) {
+            if (rtxTextureSelection.empty())
+              rtxTextureSelection = "\n";
+
+            rtxTextureSelection = str::format(rtxTextureSelection, rtxOption.displayName, "\n");
+          }
+        }
+
         char tooltip[1024];
```

Comment:
We might consider moving the tooltip outside of this scope, so we can display the same information (without the tooltip text, "right click to..." and the currently assigned catagories) in the context menu popup